### ISSUE TITLE
null-safety error on commands

### ISSUE THREAD

**Issue by EhsanAramide** [stablekernel/aqueduct#957]
I have dart 2.12.0 which is stable and support sound and unsound null safety.
When I run ```aqueduct serve``` I get an error which it says the script don't use null-safety because of this I get error and I can't do anything like migration and other commands did team have any solution for this kind of errors?

**Comment by clebsonluiz** [comment]
I have the same issue too, i can't resolve it :), btw, maybe these links can help you to get some idea,  [Another newb with null sound problems](https://github.com/stablekernel/aqueduct/issues/934) and [Support for nullsafety](https://github.com/stablekernel/aqueduct/issues/917). 

_Note *: If you don't care much about the dart sdk version, perhaps downgrading to an older version like 2.9 may allow you to continue using Aqueduct without any problems, so far this is what apparently worked for me (hehe) ._

**Comment by EhsanAramide** [comment]
yes  I did it downgrade dart sdk to 2.10 but I saw another problem in migrating that it says the value isn't in range 0...11411 do experiment it too?

**Comment by clebsonluiz** [comment]
Yep, i "fix" that looking at [this](https://github.com/stablekernel/aqueduct/issues/950), and mainly [this](https://github.com/stablekernel/aqueduct/issues/844#issuecomment-733024542).  But, what I "understood" is that apparently the problem in general is the version of the dart sdk in relation to Aqueduct and some dependences, if the Dart SDK > 2.7 some problems occur, mainly with the CLI. So, maybe if you are going to study and learn Aqueduct, using an older version of dart is the only solution so far, or waiting for an update in the future. So, sorry for not really helping because I'm also in the same boat uaheau.
